src/Extensions/Compatability/ServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+
+namespace Microsoft.Omex.Extensions.Compatability
+{
+	/// <summary>
+	/// Class to register dependencies for Compatability classes
+	/// </summary>
+	public static class ServiceCollectionExtensions
+	{
+		/// <summary>
+		/// Initialize depriacated static classes like Code and ULSLogger
+		/// </summary>
+		public static IHostBuilder AddOmexCompatabilityServices(this IHostBuilder builder) =>
```

Comment:
I don't think it will add any useful value here and in most other places. Do you think there is something that I could clarify with it? Also, I don't think it's required by ower code style.